<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelEditorQtClass</class>
 <widget class="QMainWindow" name="LevelEditorQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>400</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>11</red>
        <green>43</green>
        <blue>112</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>65</red>
        <green>29</green>
        <blue>211</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>97</red>
        <green>58</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>11</red>
        <green>43</green>
        <blue>112</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>65</red>
        <green>29</green>
        <blue>211</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>97</red>
        <green>58</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>11</red>
        <green>43</green>
        <blue>112</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>65</red>
        <green>29</green>
        <blue>211</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>97</red>
        <green>58</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>LevelEditorQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="AddEntity_Button">
       <property name="text">
        <string>Add entity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Add_EntityName_field"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="Camera_Button">
       <property name="text">
        <string>Add camera component</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="Mesh_Button">
         <property name="text">
          <string>Add mesh component</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="mesh_paht_Field"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Ligt_Button">
         <property name="text">
          <string>Add light component  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="light_intensity_Field"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>30</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> Scene name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SceneName_Field"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ENTITY:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="EntityName_Field"/>
     </item>
     <item>
      <widget class="QPushButton" name="Load_Entity_Button">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="current_entities_text">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>39</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Entities</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>41</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>247</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Transform</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Position:   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="x_pox_Field"/>
       </item>
       <item>
        <widget class="QLineEdit" name="y_pos_Field"/>
       </item>
       <item>
        <widget class="QLineEdit" name="z_pos_Field"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Rotation:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="x_rot_Field"/>
       </item>
       <item>
        <widget class="QLineEdit" name="y_rot_Field"/>
       </item>
       <item>
        <widget class="QLineEdit" name="z_rot_Field"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Scale:       </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="x_sca_Field"/>
       </item>
       <item>
        <widget class="QLineEdit" name="y_sca_Field"/>
       </item>
       <item>
        <widget class="QLineEdit" name="z_sca_Field"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Current components</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="current_components_text">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Current scene: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Current_Scene_Label">
       <property name="text">
        <string>*None*</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="set_transform_Button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set values</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionCreate_new_scene"/>
    <addaction name="actionSave_scene"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_new_scene">
   <property name="text">
    <string>Create new scene</string>
   </property>
  </action>
  <action name="actionSave_scene">
   <property name="text">
    <string>Save scene</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LevelEditorQt.qrc"/>
 </resources>
 <connections/>
</ui>
